examples/xgboost/cyclic/README.md

```diff
@@ -0,0 +1,26 @@
+# Cyclic Training of XGBoost
+
+"Cyclic XGBoost" is one way of performing federated tree boosting with multiple sites: at each round of tree boosting, instead of relying on the whole data statistics collected from all clients, the boosting relies on only 1 client's local data. The resulting tree sequence is then forwarded to the next client for next round's boosting. Such training scheme have been proposed in literatures [1] [2].
+
+### Run the experiment
+Log into the Admin client by entering `admin` for both the username and password.
+Then, use these Admin commands to run the experiment:
+
+```
+submit_job cyclic
```

Comment:
should this be submit_job "xgboost/cyclic"?

Reply:
yes updated